tools/ncbi_blast_plus/ncbi_blastp_wrapper.xml

```diff
@@ -140,10 +151,54 @@ using the NCBI BLAST+ blastp command line tool.
 
 -----
 
-@OUTPUT_FORMAT@
+**Default parameters for each type of BLAST**
```

Comment:
i don't like the idea of having to maintain a copy of the NCBI BLAST+ documentation values here - the NCBI do change defaults, and it would be all too easy for this text to get out of sync.

Reply:
Makes sense. Is a link to the command line options (https://www.ncbi.nlm.nih.gov/books/NBK279675/) better? That's what I used to make this table, but it's not clear how often they update that page.

Reply:
That sounds like a good solution to me 👍